packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -565,7 +565,7 @@ const makeOffer = async (zoe, zcf, proposal, payments) => {
 const similarToNormalZCFSeat = async (t, emptySeat, normalSeat) => {
   // Note: not exhaustive
   t.deepEqual(Object.keys(emptySeat), Object.keys(normalSeat));
-  t.deepEqual(emptySeat.getProposal(), normalSeat.getProposal());
```

Comment:
No, this is not the case we talked about. I think you are confusing two very different things. Let's call them `zcfSeatEmptySeat` and `zcfSeatMadeEarly`. `zcfSeatMadeEarly` is the task in this PR, whereas `zcfSeatEmptySeat` is the `zcfSeat` when a contract makes an offer that happens to be empty. In other words, when the contract calls `zcf.makeEmptySeatKit`, it is acting as a user and getting back a `userSeat`. These are not equivalent. `zcfSeatMadeEarly` should never result in ZCF getting a `userSeat`.

`zcfSeatEmptySeat` *has had an offer*. It has an exit rule; it just doesn't have `want` and `give`. `getProposal` for `zcfSeatEmptySeat` should not be the same as for a `zcfSeatMadeEarly` It should be:

```js
zcfSeatEmptySeat.getProposal()
// { want: {}, give: {}, exit: x }

zcfSeatMadeEarly.getProposal()
// null
```
where x is whatever `exitRule` the contract decided when it was acting as a user. 

I think given this confusion between the `zcfSeatEmptySeat` and the `zcfSeatMadeEarly`, we need more design work to ensure the flow is correct. The code that copied the flow from `makeEmptySeatKit` was incorrect, as are the changes to this file.

Reply:
I can see why having the emptySeat have an existing empty `want` and `give` fits your mental model better than it being null, but I think the way I wrote the code, it has the same effect. In both cases, we don't check offer safety. If you look at the code, I think you'll see that other places that check the proposal for one reason or another will do the right thing in both situations. Do you see any cases where one or the other has the wrong effect?

If it would be behaviorally distinct or clarifying, I can add a separate path for creating the seats, but the current approach uses a single code path that supports the behavior we need.

Reply:
Chris, this is not my mental model. A `zcfSeat` which has an offer should not have a `null` proposal. The `zcfSeat` produced by `zcf.makeEmptySeatKit()` has had an offer, therefore it should not have a `null` proposal. Please revert this. I've asked numerous times.

The functionality being equivalent is irrelevant, as this breaks a serious invariant of the design.

Reply:
If it's a "serious invariant" it should be written down somewhere if the rest of us are expected to maintain it. Did I miss a design document?

The required behavior was clear (we don't seem to disagree about that), and I ensured that that was maintained. 

I realize that you've asked numerous times, but this is the first time you've mentioned this invariant. In response to your requests, I've explained why the change maintains the required behavior. It's not like I'm ignoring you.